Core/LCG/test/Test_LCG.py

```diff
@@ -237,6 +314,25 @@ def test__downTimeXMLParsing_affected():
   res = GOCCli._downTimeXMLParsing(xml_endpoint_and_affected_ongoing, 'Site')
   assert res == {}
 
+  res = GOCCli._downTimeXMLParsing(xml_endpoint_and_affected_ongoing_broken, 'Resource')
+  assert list(res)[0] == '109962G0 lhcbsrm-kit.gridka.deSRM'
```

Comment:
Is this assuming the dictionary ordering? If so there is no guarantee that it it's stable before Python 3.6:

https://github.com/DIRACGrid/DIRAC/commit/1d2af4bcd4d253d504ae764627e8b4707de231bc#diff-67263060ea87a7fca31af7fe9ec6617c

Reply:
Well here there will be only one entry...

Reply:
I guess there will be two entries for the test that uses `xml_endpoint_and_affected_ongoing_2_endpoints`? If not, then this looks okay (though I still think `set(res) == {'109962G0 lhcbsrm-kit.gridka.deSRM'}` is a more robust check).

Reply:
@fstagni: will you update or it is OK as it is ?

Reply:
It's OK as it is.